src/qbindiff/loader/types.py

```diff
@@ -46,12 +46,14 @@ class OperandType(IntEnum):
     unknown = 0  # doc: type is unknown
     register = 1  # doc: register (GPR)
     memory = 2  # doc: Direct memory reference
-    phrase = 3  # doc: Memory access with base+reg  or  base + offset * factor
-    displacement = 4  # doc: Memory access with base+offset
-    immediate = 5  # doc: Immediate value
-    far = 6  # doc: Asbolute address
-    near = 7  # doc: Relative address
-    # TODO: To improve with architectures type specific (reglist,
+    immediate = 3  # doc: Immediate value
+    float_point = (4,)  # doc: Floating point operand
+    coprocessor = (5,)  # doc: Coprocessor operand
```

Comment:
```suggestion
    coprocessor = 5  # doc: Coprocessor operand
```

You can remove the parenthesis and just provide the value